src/libraries/Common/src/Interop/Browser/Interop.JavaScript.CoreObject.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class JavaScript
+    {
+        /// <summary>
+        /// Core objects are the standard built-in objects and functions.  These
+        /// objects are part of the JavaScript environment.  Not to be confused
+        /// with objects provided by the host application or the browser context
+        /// such as DOM.  For more information about the distinction between the
+        /// DOM and core JavaScript, see JavaScript technologies overview:
+        /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/JavaScript_technologies_overview
+        ///
+        /// Core objects are treated differently in the bridge code as they are
+        /// guaranteed to be there.
+        /// </summary>
```

Comment:
Nit: for large summaries like this, consider making the `<Summary/>` just a single short description (e.g. "Provides the base type for built-in objects.") and then move the rest of the details into a `<Remarks/>` node.

Reply:
Addressed in recent commits